src/strings/PowerFxResources.en-US.resx

```diff
@@ -4076,6 +4076,10 @@
   <data name="ErrUDF_FunctionAlreadyDefined" xml:space="preserve">
     <value>Function {0} is already defined.</value>
     <comment>This error message shows up when user tries to define a function that already exists</comment>
+  </data>
+  <data name="WrnUDF_ShadowingBuiltInFunction" xml:space="preserve">
+    <value>Warning: User defined function {0} will shadow the existing built-in function {0}.</value>
```

Comment:
>{0} [](http://example.com/codeflow?start=89&length=3)

Please add to the comment the description of the parameter and an example, such as "The argument {0} will be replaced with a function name: e.g.: User defined function Count will shadow the existing built-in function Count.`